third_party/move/move-compiler-v2/tests/bytecode-verify-failure/equality.exp

```diff
@@ -14,14 +14,10 @@ B0:
 }
 Diagnostics:
 bug: BYTECODE VERIFICATION FAILED
-  ┌─ tests/bytecode-verify-failure/equality.move:1:1
+  ┌─ tests/bytecode-verify-failure/equality.move:3:9
   │
-1 │ ╭ module 0xc0ffee::m {
-2 │ │   fun equality<T>(x: T, y: T): bool {
-3 │ │         x == y
-4 │ │     }
-5 │ │ }
-  │ ╰─^ ICE failed bytecode verifier: VMError {
+3 │         x == y
```

Comment:
I think you have also fixed this bug: https://github.com/aptos-labs/aptos-core/issues/11925. If you think so too, maybe close that bug through this PR.

Reply:
Actually not, it is only that error message is more precise because I now generate the full source map.